packages/patternfly-4/react-core/src/components/Select/Select.tsx

```diff
@@ -119,7 +121,8 @@ class Select extends React.Component<SelectProps & InjectedOuiaProps, SelectStat
     onClear: (_e: React.MouseEvent) => undefined as void,
     onCreateOption: (_newOptionValue: string) => undefined as void,
     toggleIcon: null as React.ReactElement,
-    onFilter: null
+    onFilter: null,
+    customContent: null as React.ReactNode
```

Comment:
This needs to be defined as ReactNode?

Reply:
I was just following what was done for toggleIcon.  I can change it.

Reply:
I think if we are going to change it would should have it as a coding standard.  It's fine to leave it ... however we might want to spend some time looking into our recommended coding standards so we are coding consistently through out the project.  Maybe create an issue (chore) in our backlog to update coding standards for typescript and fix the code so we are consistent.